tests/displays/cavity_display/frontend/test_fault_count_display.py

```diff
@@ -39,36 +39,34 @@ def test_get_data_no_data(qtbot: QtBot, window):
     assert window.num_invalids == []
 
 
-def test_get_data_with_pot(qtbot: QtBot, window):
+def test_get_data(qtbot: QtBot, window):
```

Comment:
Still trying to figure out tests. Do I need to make any other changes to this particular function? Do I need to generalize it to pick a random three letter fault code, or is the default 'POT' still satisfactory?